osgtest/tests/test_055_osg_ca_manage.py

```diff
@@ -0,0 +1,29 @@
+import glob
+import os
+
+import osgtest.library.core as core
+import osgtest.library.osgunittest as osgunittest
+
+
+class TestOsgCaManage(osgunittest.OSGTestCase):
+
+    def test_01_setup_ca(self):
+        """Test that a CA can be installed via `osg-ca-manage setupCA`"""
+        core.skip_ok_unless_installed('osg-ca-scripts')
+        command = ('osg-ca-manage', 'setupCA', '--location', 'root', '--url', 'osg')
+        status, stdout, stderr = core.system(command)
+        fail = core.diagnose('Run osg-ca-manage setupCA', command, status, stdout, stderr)
+
+        self.assertEquals(status, 0, fail)
+        pem_count = len(glob.glob(os.path.join('/etc/grid-security/certificates', '*.pem')))
```

Comment:
```suggestion
        pem_count = len(glob.glob('/etc/grid-security/certificates/*.pem'))
```
Don't bother doing an os.path.join on string literals, it just makes things harder to grep for.